fix(uipath-coded-apps): remove token handling guidance (#401)
UiPath/skills: skills/uipath-coded-apps/references/file-sync.md

| Variable | Description |
|----------|-------------|
| `UIPATH_PROJECT_ID` | Studio Web project ID (auto-saved on first push) |

> Authentication is handled by `uip login` and the coded app CLI. Do not read, print, source, or manually set cached access-token environment variables.

Reviewer: ```suggestion
Authentication is handled by `uip login` and the coded app CLI. **NEVER** read, print, source, or manually set cached access-token environment variables.
```
